src/src/cron/prepareCertificateHtml.cron.ts

```diff
@@ -191,22 +193,47 @@ export class PrepareCertificateHtmlCron {
 						//error in create certificate
 					}
 				}
-				// Update in attendance data in database
-				await this.markCertificateStatus(
+				// Update in certificate data in database
+				//update values
+				let testTrackingUpdateData = new Object();
+				testTrackingUpdateData['cron_last_processed_at'] = new Date(
+					Date.now(),
+				).toISOString();
+				testTrackingUpdateData['attendance_count'] =
+					usrAttendanceList.length;
+				if (issue_status == 'true') {
+					testTrackingUpdateData['certificate_status'] = issue_status;
+				}
+				await this.updateTestTrackingData(
 					userTestData?.id,
-					issue_status,
+					testTrackingUpdateData,
 				);
 			}
 		}
 	}
 	async fetchTestTrackingData(limit: number) {
+		// We need to skip processing records wch were processed in past X hours
+		let dt = new Date();
+		let filterTimestamp = new Date(
+			dt.setHours(dt.getHours() - 4),
```

Comment:
Get this 4 from config / env

Reply:
2 is hours limit for last_processed. just like 4 we used in attedance face rekognition.
after testing done on dev I will move it in env variba